CRM/Report/Form/Instance.php

```diff
@@ -222,7 +222,7 @@ public static function setDefaultValues(&$form, &$defaults) {
 
     if (!isset($defaults['permission'])) {
       $permissions = array_flip(CRM_Core_Permission::basicPermissions());
-      $defaults['permission'] = $permissions['CiviReport: access CiviReport'];
+      $defaults['permission'] = $permissions['CiviReport: ' . ts('access CiviReport')];
```

Comment:
The above would fail if the CiviReport is also translated.
IMO it would be better if we rely on untranslated format
`if (!isset($defaults['permission'])) {
     $defaults['permission'] = 'access CiviReport';
}`